omod/src/main/webapp/pages/conditionlist/addCondition.gsp

```diff
@@ -60,21 +60,21 @@
                         endDate    : new Date(),
                 ])}
             </li>
+            <div id="status" class="horizontal" style="margin-top:10px;">
```

Comment:
Is this a formatting change? Are all these changes necessary? Can we just have only what you changed?

Reply:
Yes, the changes are necessary to get the alignment as the position for the element is relative.